content/en/docs/components/katib/trial-template.md

```diff
@@ -1,13 +1,13 @@
 +++
 title = "Trial Template Overview"
 description = "How to specify trial template parameters and support custom CRD in Katib"
```

Comment:
```suggestion
description = "How to specify trial template parameters and support a custom resource (CRD) in Katib"
```

(I had to look up CRD on the K8s docs - correct me if I'm wrong. Just wanted to help some new users)